pkg/debuginfo/manager_test.go

```diff
@@ -76,3 +76,36 @@ func BenchmarkEnsureUploadedInitiateUploadError(b *testing.B) {
 		require.Equal(b, codes.Internal, status.Code(errors.Unwrap(err)))
 	}
 }
+
+func TestHasTextSection(t *testing.T) {
+	testCases := []struct {
+		name              string
+		filepath          string
+		textSectionExists bool
+		wantErr           bool
+	}{
+		{
+			name:              "text section present",
+			filepath:          "./testdata/elf-file-with-text-section",
```

Comment:
For this, we don't need to add an extra file, let's reuse one of the existing test data files. Like redelf-sections

Reply:
thanks for pointing it out, done.